packages/components/src/navigator/styles.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * External dependencies
+ */
+import { css, keyframes } from '@emotion/react';
+
+export const navigatorProviderWrapper = css`
+	/* Prevents horizontal overflow while animating screen transitions */
+	overflow-x: hidden;
+	/* Mark this subsection of the DOM as isolated, providing performance benefits
+	 * by limiting calculations of layout, style, paint, size, or any combination
+	 * to a DOM subtree rather than the entire page.
+	 */
+	contain: strict;
```

Comment:
The `contain: strict` makes everything (that doesn't have height or isn't inside a flex with grow) have 0 height for me. I use Navigator in 4 places and all of them "broke", (which is fine, since it's __experimental), but just an FYI.

Reply:
Thank you for letting us know! The fix has been merged in #57498